core/trino-main/src/test/java/io/trino/execution/BaseDataDefinitionTaskTest.java

```diff
@@ -122,6 +123,14 @@ public void setUp()
         queryStateMachine = stateMachine(transactionManager, createTestMetadataManager(), new AllowAllAccessControl(), testSession);
     }
 
+    @AfterMethod
+    public void tearDown()
```

Comment:
Make final so people can't accidentally override it. And maybe name it `tearDownBaseTest`. Same for the setup method.